sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/EncryptionKeyWrapMetadata.java

```diff
@@ -41,15 +42,18 @@ public EncryptionKeyWrapMetadata(EncryptionKeyWrapMetadata source) {
      * @param type Type of the metadata.
      * @param name Name of the metadata.
      * @param value Value of the metadata.
+     * @param algorithm Algorithm of the metadata.
      */
     @Beta(value = Beta.SinceVersion.V4_16_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
-    public EncryptionKeyWrapMetadata(String type, String name, String value) {
+    public EncryptionKeyWrapMetadata(String type, String name, String value, String algorithm) {
```

Comment:
This will always fail the rev-API check. We might need to deprecate this, and add a new constructor to avoid this. This also enables us to remove this constructor in a couple of releases.

Reply:
done